python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -3,99 +3,285 @@
+                    'libpcre.so.1',
+                ])
+    elif platform_system == 'aix':
+        # This is the standard list of deps for AIX 5.3. Some of the links
+        # for these libs moved in newer versions from '/usr/lib/' to '/lib/'.
+        allowed_deps = [
+            '/unix',
+            'libbsd.a',
+            'libc.a',
+            'libcrypt.a',
+            'libcrypto.a',
+            'libdl.a',
+            'libnsl.a',
+            'libpthreads.a',
+            'libpthreads_compat.a',
+            'libssl.a',
+            'libtli.a',
+            'libz.a',
+            ]
+        # sys.platform could be 'aix5', 'aix6' etc.
+        aix_version = int(sys.platform[-1])
+        if aix_version >= 7:
+            allowed_deps.extend([
+                'libthread.a',
+            ])
+    elif platform_system == 'sunos':
+        # This is the standard list of deps for a Solaris 10 build.
+        # For now, we include the major versions for Solaris libs.
+        allowed_deps = [
+            'libaio.so.1',
+            'libc.so.1',
+            'libcrypt_i.so.1',
+            'libcrypto.so.0.9.7',
+            'libcrypto_extra.so.0.9.7',
+            'libdl.so.1',
+            'libdoor.so.1',
+            'libgen.so.1',
+            'libintl.so.1',
+            'libm.so.2',
+            'libmd.so.1',
+            'libmp.so.2',
+            'libnsl.so.1',
+            'librt.so.1',
+            'libscf.so.1',
+            'libsocket.so.1',
+            'libsqlite3.so.0',
+            'libssl.so.0.9.7',
+            'libssl_extra.so.0.9.7',
+            'libuutil.so.1',
+            'libz.so.1',
+            ]
+    elif platform_system == 'darwin':
+        # This is the minimum list of deps for OS X.
+        allowed_deps = [
+            'ApplicationServices.framework/Versions/A/ApplicationServices',
+            'Carbon.framework/Versions/A/Carbon',
+            'CoreFoundation.framework/Versions/A/CoreFoundation',
+            'CoreServices.framework/Versions/A/CoreServices',
+            'SystemConfiguration.framework/Versions/A/SystemConfiguration',
+            'libSystem.B.dylib',
+            'libcrypto.0.9.8.dylib',
+            'libgcc_s.1.dylib',
+            'libssl.0.9.8.dylib',
+            'libz.1.dylib',
+            ]
+    return allowed_deps
+
+
+def get_actual_deps():
+    """
+    Return the list of actual deps for the newly-built binaries.
+    """
     try:
-        from ctypes import windll
-        windll
+        raw_deps = subprocess.check_output(script_helper).splitlines()
     except:
-        print '"ctypes - windll" missing.'
-        exit_code = 1
+        sys.stderr.write('Could not get the deps for the new binaries.\n')
+    else:
+        libs_deps = []
+        for line in raw_deps:
+            # We exclude any occuring lines listing the examined files as we
+            # know they start with './' (this is a problem in AIX and OS X).
+            if line[:2] != './' :
```

Comment:
Fine by me… I'll also try to rephrase the comment to make it clear what is this all about.